ui/src/app/components/multiple-choice-question.tsx

```diff
@@ -22,6 +22,7 @@ const styles = reactStyles({
     color: colors.primary,
     fontSize: '18px',
     lineHeight: '1.25rem',
+    marginBottom: '0.5rem',
```

Comment:
This also changes the Demo Survey's appearance, and I don't think we want that.  For situations like these, an optional style prop works well.  We have this already for the container, and we could do something similar.  Maybe like this:
```
  return (
    <div style={{ ...styles.container, ...style }}>
      <div style={{ ...styles.question, ...questionStyle }}>{question}</div>

```

Reply:
[ok now I see you mentioned this.  Yeah I think I prefer the original style for Demo Survey, sorry]

Reply:
Moved into a prop like you suggested. I also removed this change from the PR description.